samples/System.IO.Pipelines.Samples/CompressionSample.cs

```diff
@@ -14,38 +14,35 @@ public class CompressionSample : ISample
     {
         public Task Run()
         {
-            using (var bufferPool = new MemoryPool())
-            {
-                var filePath = Path.GetFullPath("Program.cs");
+            var filePath = Path.GetFullPath("Program.cs");
```

Comment:
Use of var here and elsewhere

Reply:
?

Reply:
State type explicitly where it isn't statically clear from the right hand side of the expression, what the type is. Only use vars in those scenarios.

Reply:
Use of var, I read "Use var". Will do.